optuna/integration/cma.py

```diff
@@ -317,8 +316,11 @@ def __init__(
                 lows.append(0 - 0.5 * dist.q)
                 highs.append(r + 0.5 * dist.q)
             elif isinstance(dist, IntUniformDistribution):
-                lows.append(dist.low - 0.5)
-                highs.append(dist.high + 0.5)
+                lows.append(dist.low - 0.5 * dist.step)
+                highs.append(dist.high + 0.5 * dist.step)
```

Comment:
Just wondering but is this change somewhat orthogonal to the other?

Reply:
You're right. It's the change for `IntUniformDistribution`.  I reverted the change in https://github.com/optuna/optuna/pull/1302/commits/682e8f4fac1f5e7a22af99cb93957041681815e9. Thank you for your careful review.

Reply:
Should we create a separate PR for that?

Reply:
Thank you for your suggestion.
I created #1456. Please review it after this PR.

Reply:
Thanks, I'll have a look at it.